### ISSUE TITLE
Control update versioning logic

### ISSUE THREAD

**Issue by MM-Lehmann** [samply/blaze#680]
We have a very limited storage on the server blaze runs on. From time to time, disk space runs out and blaze exhibits a weird behaviour, clearly not being aware of the underlying problem. We suspect there isn't enough space to run the indexing.
One possible solution we want to explore is reducing the amount of data that is written to the server every day:
We are running an ETL job once a day which publishes ~40k bundles containing ~60k resources in total. As resource versioning is enabled on the server, this results in a new version every day, for some resources even more due to the nature of our dataset.
It would be the easiest and suffice our use case, if there was a way to disable versioning on the server altogether. We make a full load every day and don't really need that feature. 
How can we achieve that? Is there a hidden feature flag somewhere?

**Comment by alexanderkiel** [comment]
Unfortunately there is currently no way to disable versioning. We have rough plans to remove the index entries of old versions, because queries on old versions are not part of the FHIR API. But this is currently not a high priority.

Blaze needs from time-to-time double the amount of disk space to do compactions of the database storage. So clearly, Blaze trades disk space for correctness and speed. So saving disk space will not be a priority for the Blaze development.

**Comment by MM-Lehmann** [comment]
Would it be in-scope that blaze checks on available disk space before any task and reports this as an error either on the server or (even better) as an http reponse? It took us a while to figure out what the problem really was...

**Comment by alexanderkiel** [comment]
I'll close this issue because #1041 now dosen't use up much additional disk space on resource updates without changes.
